index.js

```diff
@@ -283,10 +284,12 @@ AFRAME.registerComponent('teleport-controls', {
       if (!this.data.cameraRig) {
         var hands = document.querySelectorAll('a-entity[tracked-controls]');
         for (var i = 0; i < hands.length; i++) {
-          var position = hands[i].getAttribute('position');
-          var diff = auxVector3.copy(this.rigWorldPosition).sub(position);
-          var newPosition = auxVector3.copy(this.newRigWorldPosition).sub(diff);
```

Comment:
@netpro2k I didn't spot the problem when I just reviewed the code without testing it. As you're using auxVector3 reference in the diff but before that it's being updated as it's used also for copy newRigWorldPosition, the newPosition is always (0,0,0)

Reply:
Looks like this was introduced in a later commit here https://github.com/fernandojsg/aframe-teleport-controls/commit/1f1cc6a7f41d6a702be0e6e16e936581066cef41#diff-168726dbe96b3ce427e7fedce31bb0bcR267 the original was using 2 different vectors. Probably due to the confusing re-use of the rig vectors on my part though.

Reply:
LoL my fault xD

Reply:
In any case It was not a blame, i Hope you didnt take It that way :P i was so happy to catch that nasty bug that wanted yo share